native/package.json

```diff
@@ -97,6 +97,7 @@
     "@testing-library/jest-native": "^3.4.3",
     "@testing-library/react-native": "^7.1.0",
     "@types/hashids": "^1.0.32",
+    "@types/htmlparser2": "^3.10.2",
```

Comment:
I guess we can remove `htmlparser2` and its types from both the native and web package.json now, right?